episodes/quantify-transmissibility.Rmd

```diff
@@ -0,0 +1,416 @@
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: prereq
+
+## Prerequisites
+
+Learners should familiarise themselves with following concept dependencies before working through this tutorial: 
+
+**Statistics** : probability distributions, principle of Bayesian analysis. 
+
+**Epidemic theory** : Effective reproduction number.
+
+:::::::::::::::::::::::::::::::::
+
+
+
+::::::::::::::::::::::::::::::::::::: callout
+### Reminder: the Effective Reproduction Number, $R_t$ 
+
+The [basic reproduction number](../learners/reference.md#basic), $R_0$, is the average number of cases caused by one infectious individual in a entirely susceptible population. 
+
+But in an ongoing outbreak, the population does not remain entirely susceptible as those that recover from infection are typically immune. Moreover, there can be changes in behaviour or other factors that affect transmission. When we are interested in monitoring changes in transmission we are therefore more interested in the value of the **effective reproduction number**, $R_t$, the average number of cases caused by one infectious individual in the population at time $t$.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Introduction
+
+Quantifying transmission metrics at the start of an outbreak can give important information on the strength of transmission (reproduction number) and the speed of transmission ([growth rate](../learners/reference.md#growth), doubling/halving time). To estimate these key metrics using case data we must account for delays between the date of infections and date of reported cases. In an outbreak situation, data are usually available on reported dates only, therefore we must use estimation methods to account for these delays when trying to understand changes in transmission over time. 
+
+In the next tutorials we will focus on how to implement the functions in `{EpiNow2}` to estimate transmission metrics of case data. We will not cover the theoretical background of the models or inference framework, for details on these concepts see the [vignette](https://epiforecasts.io/EpiNow2/dev/articles/estimate_infections.html).
+For more details on the distinction between speed and strength of transmission and implications for control, see [Dushoff & Park, 2021](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2020.1556).
+
+
+::::::::::::::::::::::::::::::::::::: callout
+### Bayesian inference
+
+The R package `EpiNow2` uses a [Bayesian inference](../learners/reference.md#bayesian) framework to estimate reproduction numbers and infection times based on reporting dates.
+
+In Bayesian inference, we use prior knowledge (prior distributions) with data (in a likelihood function) to find the posterior probability.
+
+<p class="text-center" style="background-color: white">Posterior probability $\propto$ likelihood $\times$ prior probability
+</p>
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
```

Comment:
@jamesmbaazam requested to clarify the wording from "posterior probability" to "posterior distribution". The current phrasing looks consistent with documentation found elsewhere ([wikipedia](https://en.wikipedia.org/wiki/Posterior_probability#Definition_in_the_distributional_case), [nature methods](https://www.nature.com/articles/nmeth.3368)). 

I suggest adding an instructor callout after this, to make this clarification explicit. Also relate this with the usage of "posterior probability" in the "expected change on daily cases" callout just before the [Quantify geographical heterogeneity](https://epiverse-trace.github.io/tutorials-middle/instructor/quantify-transmissibility.html#quantify-geographical-heterogeneity) section. 

```suggestion

:::::::::::::::::::::::::::::::::::::::::::::::: instructor

We refer to the Prior probability distribution and the [Posterior probability](https://en.wikipedia.org/wiki/Posterior_probability) distribution.

Lines below, in the "`Expected change in daily cases`" callout, by "the posterior probability that $R_t < 1$", we refer specifically to the area under the posterior probability distribution curve. 

::::::::::::::::::::::::::::::::::::::::::::::::

```